.github/workflows/main-build.yml

```diff
@@ -237,6 +237,15 @@ jobs:
       appsignals-adot-image-name: ${{ needs.build.outputs.staging-image }}
       caller-workflow-name: 'main-build'
 
+  # AppSignals specific e2e tests for ec2
+  appsignals-e2e-ec2-test:
+    needs: build
+    uses: ./.github/workflows/appsignals-e2e-eks-test.yml
+    secrets: inherit
+    with:
+      aws-region: ${{ needs.build.outputs.aws_default_region }}
+      caller-workflow-name: 'main-build'
+
   publish-build-status:
```

Comment:
Curious as to why do we need to publish status metric for the main build? Are we monitoring this metric somewhere?

Reply:
I'm not too sure. The e2e-eks testing is publishing the status so I did the same for ec2